difflint/lib.py

```diff
@@ -13,6 +14,29 @@
 MISSING_FILE_EXIT_CODE = 72  # os.EX_OSFILE is not portable
 
 
+def make_path_from_cwd(*args):
+    """
```

Comment:
nit: rm extra newline after `"""` for pep8 compliance. Apparently there are tools that parse python documentation that rely on the assumption that the first line of the function comment gives a summary of its behavior.